lms/djangoapps/mobile_api/groups/views.py

```diff
@@ -230,12 +40,17 @@ class GroupsCreate(generics.CreateAPIView):
     permission_classes = (permissions.IsAuthenticated,)
     
     def create(self, request, *args, **kwargs):
```

Comment:
Just in reference to my last comment above --- this would be an example of a function you would override with a multi-operation APIView. You could use RetrieveUpdateDestroyAPIView, and override create/delete functions (which I believe exist, but i'm not sure are necessarily named as such).